db/migrate/20221108062525_add_index_for_discord_user_id.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexForDiscordUserId < ActiveRecord::Migration[6.1]
+  def change
+    add_index :users, :discord_user_id, unique: true
```

Comment:
This probably shouldn't have `unique: true` on it. It should be possible for users from different schools to have the same `discord_user_id`. If you want a uniqueness constraint, set up a composite key:

```rb
add_index :users, [:school_id, :discord_user_id], unique: true
```

The `User` model should also be updated to inform ActiveRecord of the constraint.